crates/fuel-core/src/state.rs

```diff
@@ -68,25 +65,12 @@ pub trait BatchOperations: KeyValueStore {
 
 #[derive(Debug)]
 pub enum WriteOperation {
-    Insert(Vec<u8>, Column, Vec<u8>),
-    Remove(Vec<u8>, Column),
+    Insert(Value),
```

Comment:
Do we still need to have all these changes to the write operation given that we are now using the built-in row-cache for rocksdb?